lib/components/component.dart

```diff
@@ -16,6 +17,9 @@ abstract class Component {
   /// HUD objects ignore the [BaseGame.camera] when rendered (so their position coordinates are considered relative to the device screen).
   bool isHud = false;
 
+  /// Whether this component is currently mounted on a game or not
+  bool isMounted = false;
```

Comment:
What do you think about this name? Is `isAttached` a better name, since this is set to false when it is removed from the game? This is however more clear that it is set by `onMount`...

Reply:
tbh, IMO `isMounted` is fine, but I have nothing against `isAttached` as well hahha

Reply:
I personally would say `isMounted`, considering that is what it is indicating. It is either mounted to a game or not. And the methods that can influence it are also using the same naming convention :-).

I do have a quick wonder, what would happen if I as a user update this value? Would that do anything at all? If it doesn't should this not be a `getter`, and only let the system(flame) control the value?

Reply:
@wolfenrain good point, did that change now.